docs/infra/intro-to-terraform-workspaces.md

```diff
@@ -1,6 +1,6 @@
 # Workspaces
 
-Terraform workspaces are created by default, the default workspace is named "default." Workspaces are used to allow multiple engineers to deploy their own stacks for development and testing. This allows multiple engineers to develop new features in parallel using a single environment without destroying each others infrastructure. Separate resources will be created for each engineer when using the prefix variable.
+Terraform workspaces are created by default, the default workspace is named "default." Workspaces are used to allow multiple engineers to deploy their own stacks for development and testing. This allows multiple engineers to develop new features in parallel using a single environment without destroying each others' infrastructure. Separate resources will be created for each engineer when using the prefix variable.
```

Comment:
nit: Turns out, it's singular. `each other's`. You can play around mentally with `each [something]` as the subject of a sentence and see it uses a singular verb conjugation. But I check with google as well. (E.g., "each dog eats" as in "a single dog eats" and not "each dog eat" like "all the dogs eat". It's harder to do such comparisons with the possessive.)

![image](https://github.com/navapbc/template-infra/assets/5387486/3278bfd1-86a0-4026-938e-84dcf00a840a)